src/activate/activate.c

```diff
@@ -385,6 +396,11 @@ static int parse_argv(int argc, char *argv[]) {
                 return -EINVAL;
         }
 
+        if (arg_datagram && arg_accept) {
+                log_error("Datagram sockets do not accept connections. Use -a option only with stream sockets.");
```

Comment:
Sorry, the second sentence isn't correct English. Please reword to "The -a option can only be used with stream sockets.". Thanks!